src/python/aqueduct_executor/operators/function_executor/execute.py

```diff
@@ -40,17 +40,32 @@ def _get_py_import_path(spec: FunctionSpec) -> str:
 
     if file_path.startswith("/"):
         file_path = file_path[1:]
-    return ".".join([OP_DIR, file_path.replace("/", ".")])
+    return file_path.replace("/", ".")
 
 
 def _import_invoke_method(spec: FunctionSpec) -> Callable[..., DataFrame]:
+    # `_import_invoke_method` imports the model object.
+    # it assumes the operator has been extracted to `<storage>/operators/<id>/op`
+    # and imports the route from the above path.
+
+    # fn_path should be `<storage>/operators/<id>`
     fn_path = spec.function_extract_path
-    os.chdir(os.path.join(fn_path, OP_DIR))
-    sys.path.append(fn_path)
+
+    # work_dir should be `<storage>/operators/<id>/op`
+    work_dir = os.path.join(fn_path, OP_DIR)
+    print("listdir workdir")
```

Comment:
Let's be a bit more descriptive here: "Listing files under the current working directory." The reason being I feel others may have a hard time understanding what `listdir workdir` means.

Reply:
I agree. I think it is kind of confusing what listdir and workdir represent, might need some documentation or clarification there.